### PR TITLE
Revise Versioning Structure v2 --> v2.3

### PR DESCRIPTION
**Pull request by nate-double-u**
Moving files. 

Updating config.toml, partials/docs/hero.html, partials/docs/nav-panel.html, and content/docs/v2.3/_index.md

Contributes to: https://github.com/etcd-io/website/issues/152

/cc @chalin

### CODE REVIEW

**Comment by nate-double-u** [comment]
deploy preview: https://deploy-preview-154--etcd.netlify.app/

**Comment by chalin** [comment]
And the `_redirects`?

**Comment by nate-double-u** [comment]
> And the `_redirects`?

Right!

**Comment by nate-double-u** [comment]
@chalin, I've added the `static/_redirects` file, and the redirect rules check is coming back passed, but the deploy preview doesn't seem to actually redirect. 

https://deploy-preview-154--etcd.netlify.app/docs/v2/ <- should show https://deploy-preview-154--etcd.netlify.app/docs/v2.3/

Thoughts?

**Comment by nate-double-u** [comment]
actually, the redirect check is still only checking 4 rules, so netlify isn't seeing `static/_redirects`

**Comment by chalin** [comment]
I know that it isn't obvious, but the file to change is https://github.com/etcd-io/website/blob/master/layouts/index.redirects. Sorry for not being clearer earlier, but your name shows up in the changes to that file so I thought that you knew / remembered.

**Comment by chalin** [comment]
(It's the _generated_ file that is named `_redirects`.)

**Comment by nate-double-u** [comment]
I've tried updating `index.redirects` as follows:

```
/docs/latest     /docs/v3.4.0
/docs/latest/*     /docs/v3.4.0/:splat
/docs/v2/*    /docs/v2.3/:splat 301!
```
which generates a `_redirects` file of: 
```

/docs/latest     /docs/v3.4.0
/docs/latest/*     /docs/v3.4.0/:splat
/docs/v2/*    /docs/v2.3/:splat 301!
```
But this doesn't actually forward the url (at least with `netlify dev`)

Updating `netlfy.toml` with:
```
[[redirects]]
from = "/docs/v2/*"
to = "/docs/v2.3/:splat"
status = 301
force = true
```
only works (with `netlify dev` if you use a trailing slash).

File: netlify.toml [hunk 1]
```diff
@@ -33,3 +33,8 @@ to = "/docs/v3.4.0/reporting_bugs"
 status = 301
 force = false
 
+[[redirects]]
+from = "/docs/v2/*"
+to = "/docs/v2.3/:splat"
+status = 301
+force = true
```
Comment (chalin) [inline netlify.toml:40]: You shouldn't need froce.

**Comment by nate-double-u** [comment]
OK, testing the `netlify.toml` change:
https://605bb02c277eb10007b716ee--etcd.netlify.app/docs/v2/
https://605bb02c277eb10007b716ee--etcd.netlify.app/docs/v2
(trailing slash, no trailing slash)
Both work.

Next, testing just updating the `index.redirects` file. (leaving this one for test 2, so if it works we can just leave it. Sorry for the PR spam. It looks like `netlify dev` behaves a little differently than Netlify does)

File: layouts/index.redirects [hunk 2]
```diff
@@ -1,3 +1,4 @@
 {{ $latest := site.Params.versions.latest }}
 /docs/latest     /docs/{{ $latest }}
 /docs/latest/*     /docs/{{ $latest }}/:splat
+/docs/v2/*    /docs/v2.3/:splat 301
```
Comment (chalin) [inline layouts/index.redirects:4]: You don't need the 301.

**Comment by nate-double-u** [comment]
Testing the `index.redirects` file, both
https://605bb1b1e3f1c200072c1494--etcd.netlify.app/docs/v2
https://605bb1b1e3f1c200072c1494--etcd.netlify.app/docs/v2/

work.

File: layouts/index.redirects (hunk 2 above)
Comment (chalin) [inline layouts/index.redirects]: And you'll need two rules. One for `/docs/v2` and the other that you have here.

**Comment by nate-double-u** [comment]
we'll squash this PR 😅

File: layouts/index.redirects [hunk 3]
```diff
@@ -1,3 +1,5 @@
 {{ $latest := site.Params.versions.latest }}
 /docs/latest     /docs/{{ $latest }}
 /docs/latest/*     /docs/{{ $latest }}/:splat
+/docs/v2    /docs/v2.3/:splat
```
Comment (chalin) [inline layouts/index.redirects:4]: No splat in this rule:

     /docs/v2 /docs/v2.3

File: layouts/index.redirects (hunk 2 above)
Comment (chalin) [reply layouts/index.redirects]: FYI, AFAIR, rules starting with `/a/b/c` or `/a/b/c/` are the same.

**Comment by nate-double-u** [comment]
OK, i think that's got it

https://deploy-preview-154--etcd.netlify.app/docs/v2
https://deploy-preview-154--etcd.netlify.app/docs/v2/

File: layouts/index.redirects [hunk 4]
```diff
@@ -1,3 +1,5 @@
 {{ $latest := site.Params.versions.latest }}
 /docs/latest     /docs/{{ $latest }}
 /docs/latest/*     /docs/{{ $latest }}/:splat
+/docs/v2    /docs/v2.3
```
Comment (chalin) [inline layouts/index.redirects:4]: The rules are working. One last change: add a trailing slash to the end of this rule.

Comment (chalin) [reply layouts/index.redirects:4]: Oh, and to the rule in line 2 as well :).

**Review by chalin — APPROVED** [review]
LGTM 🎉

**Pull request merged**